webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -13,6 +13,15 @@
 
             .section-body {
                 padding: 32px 48px 48px;
+
+                // Custom components are sometimes nested, but don't do their horizontal spacing again
+                .section-header {
+                    padding: 48px 0 0;
+                }
+            }
+
+            .custom-section-body {
+                padding: 32px 0 0;
```

Comment:
Personally, I find these directives a little confusing because I need to find a mental map of the syntax. A ` top | left and right | bottom` comment would be nice.